### ISSUE TITLE
Saving attributes throws error in version `2.8.15`

### ISSUE THREAD

**Issue by doishub** [isotope/core#2480]
With the latest version, errors are thrown when saving attributes:
```
An exception occurred while executing a query: SQLSTATE[22007]: Invalid datetime format: 1292 Truncated incorrect DOUBLE value: ''
```

**Comment by aschempp** [comment]
Please post the full stack trace.

**Comment by doishub** [comment]
```
Doctrine\DBAL\Exception\DriverException:
An exception occurred while executing a query: SQLSTATE[22007]: Invalid datetime format: 1292 Truncated incorrect DOUBLE value: ''

  at C:\...\vendor\doctrine\dbal\src\Driver\API\MySQL\ExceptionConverter.php:117
  at Doctrine\DBAL\Driver\API\MySQL\ExceptionConverter->convert(object(Exception), object(Query))
     (C:\...\vendor\doctrine\dbal\src\Connection.php:1935)
  at Doctrine\DBAL\Connection->handleDriverException(object(Exception), object(Query))
     (C:\...\vendor\doctrine\dbal\src\Connection.php:1878)
  at Doctrine\DBAL\Connection->convertExceptionDuringQuery(object(Exception), 'UPDATE `tl_iso_product` SET `product`=0 WHERE `product`=\'\'', array(), array())
     (C:\...\vendor\doctrine\dbal\src\Connection.php:1103)
  at Doctrine\DBAL\Connection->executeQuery('UPDATE `tl_iso_product` SET `product`=0 WHERE `product`=\'\'', array(), array())
     (C:\...\vendor\contao\core-bundle\src\Resources\contao\library\Contao\Database\Statement.php:284)
  at Contao\Database\Statement->query('UPDATE `tl_iso_product` SET `product`=0 WHERE `product`=\'\'')
     (C:\...\vendor\contao\core-bundle\src\Resources\contao\library\Contao\Database.php:224)
  at Contao\Database->query('UPDATE `tl_iso_product` SET `product`=0 WHERE `product`=\'\'')
     (C:\...\vendor\isotope\isotope-core\system\modules\isotope\library\Isotope\DatabaseUpdater.php:47)
  at Isotope\DatabaseUpdater->autoUpdateTables(array('tl_iso_product'))
     (C:\...\vendor\isotope\isotope-core\system\modules\isotope\library\Isotope\Backend\Attribute\DatabaseUpdate.php:48)
  at Isotope\Backend\Attribute\DatabaseUpdate->onSubmit(object(DC_Table))
     (C:\...\vendor\contao\core-bundle\src\Resources\contao\drivers\DC_Table.php:2128)
  at Contao\DC_Table->edit()
     (C:\...\vendor\isotope\isotope-core\system\modules\isotope\library\Isotope\BackendModule\BackendOverview.php:245)
  at Isotope\BackendModule\BackendOverview->getModule('attributes')
     (C:\...\vendor\isotope\isotope-core\system\modules\isotope\library\Isotope\BackendModule\BackendOverview.php:84)
  at Isotope\BackendModule\BackendOverview->generate()
     (C:\...\vendor\contao\core-bundle\src\Resources\contao\classes\Backend.php:439)
  at Contao\Backend->getBackendModule('iso_setup', null)
     (C:\...\vendor\contao\core-bundle\src\Resources\contao\controllers\BackendMain.php:168)
  at Contao\BackendMain->run()
     (C:\...\vendor\contao\core-bundle\src\Controller\BackendController.php:49)
  at Contao\CoreBundle\Controller\BackendController->mainAction()
     (C:\...\vendor\symfony\http-kernel\HttpKernel.php:163)
  at Symfony\Component\HttpKernel\HttpKernel->handleRaw(object(Request), 1)
     (C:\...\vendor\symfony\http-kernel\HttpKernel.php:75)
  at Symfony\Component\HttpKernel\HttpKernel->handle(object(Request), 1, true)
     (C:\...\vendor\symfony\http-kernel\Kernel.php:202)
  at Symfony\Component\HttpKernel\Kernel->handle(object(Request))
     (C:\...\public\index.php:44)                
```

**Comment by aschempp** [comment]
What kind of attribute did you create?

**Comment by doishub** [comment]
Here is a screenshot of the `product` attribute. However, the error occurs in every attribute (regardless of the settings):
![image](https://github.com/isotope/core/assets/48379929/319422dd-8413-406d-a07a-fa4702cf521d)

When I reload the page after the error, it has also taken over everything and it works as desired. But the error occurs every time I save.

I had already thought whether "product" might be a blocked column name, but I couldn't change it because the shop is already set up to that extent and everything references it. However, the error would probably have appeared in the last version.

**Comment by aschempp** [comment]
It does not necessarily have to do with this attribute, because all DB changes are applied when an attribute is saved. 

Can you maybe check the install tool or Contao Manager what field exactly is tried to be changed?

**Comment by doishub** [comment]
No updates are displayed there after the error message. 🤔

**Comment by Weuza84** [comment]
I get the same exception, when updating an attribute:

An exception occurred while executing a query: SQLSTATE[22007]: Invalid datetime format: 1292 Truncated incorrect DECIMAL value: ''

**Comment by aschempp** [comment]
Fixed in 5367de65f74cff7d015432d07e3a8da4d5fc0636
